superset-installer/etc/reflex-provisioner/inventory/templates/group_vars/global/all/raf/superset.yml

```diff
@@ -90,7 +90,7 @@ superset_krb5_conf_name: "krb5.conf"
 
 # Config Diretory
 superset_config_dir: "{{ role_path }}/../kerberos/files/config"
-superset_kerberos_enabled: false
+superset_kerberos_enabled: true
```

Comment:
@ankursinghal2005 this we should keep as default true in new ansible role as well.

Reply:
Yes, security should be enabled by default. Already take care in new anisble role..